### PR TITLE
♻️ Expose Gestures methods publicly

### PR DESCRIPTION
**Pull request by jridgewell**
They're used by an external caller, so not private.

Part of #14761.

### CODE REVIEW

File: src/gesture.js [hunk 1]
```diff
@@ -323,9 +323,8 @@ export class Gestures {
    * recognizer to proceed.
    * @param {!GestureRecognizer} recognizer
    * @param {number} offset
-   * @private
    */
-  signalReady_(recognizer, offset) {
+  signalReady(recognizer, offset) {
```
Comment (rsimha) [inline src/gesture.js]: Shouldn't the name be changed at the external call sites too?

Comment (jridgewell) [reply src/gesture.js]: https://github.com/ampproject/amphtml/pull/14905/files#diff-971cca60811e44231f07a088cc5bc30fR584

Comment (rsimha) [reply src/gesture.js]: Aha, external to the class, internal to the file.

Comment (jridgewell) [reply src/gesture.js]: What do you think about defining `@friend` that would expose these to others in the file? It wouldn't help for truly external callers, but some of these classes really are tied closely.

Comment (rsimha) [reply src/gesture.js]: Would this work? http://usejsdoc.org/tags-protected.html

**Pull request closed without merging**